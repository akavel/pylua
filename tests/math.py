print 3**2
print 9/3
print 4*7
print 2+3
print 10-2
print 8%3

