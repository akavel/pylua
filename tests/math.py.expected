9
3
28
5
8
2

